chains/arcadia/metadata.yaml

```diff
@@ -14,7 +14,8 @@ deployer:
   url: https://www.hyperlane.xyz
 displayName: Arcadia
 domainId: 4278608
-gasCurrencyCoinGeckoId: khalani-aip # TODO: update once we have a valid id
+# TODO: update to khalani-aip once we have a valid id
+gasCurrencyCoinGeckoId: ethereum
```

Comment:
curious why ethereum? is AIP close to ETH in price?

Reply:
in the tokenPrices.json we've always been using the eth price - this just reflects that directly in registry. there's some aip/eth redemption mechanism iirc which is why we chose eth